src/bp-members/bp-members-functions.php

```diff
@@ -4853,15 +4853,18 @@ function bb_is_online_user( $user_id, $expiry = false ) {
 	}
 
 	if ( true === $expiry ) {
```

Comment:
it should be if $expiry is empty. then use bb_presence_interval(). else use the time provided by $expiry variable.